### ISSUE TITLE
Visual regression updates

### ISSUE THREAD

**Issue by mcoker** [patternfly/patternfly#6602]
* @wise-king-sullyman  added a [`.page-loaded` class to `<body>`](https://github.com/patternfly/patternfly-org/pull/3789/files) once a full page example loads that we should use as backstop's `readySelector` https://github.com/patternfly/patternfly/blob/cd1e552eede3f3b55f4f40a2cce5ce830b48b08c/backstop.js#L12
* We could look at adding a screenshot job for dark theme! That may be a separate issue unless it's really straight forward.

**Comment by wise-king-sullyman** [comment]
When doing this issue we'll probably want to go back before at least https://github.com/patternfly/patternfly/pull/6598 and generate the base dark theme screenshots from then. #6598 included a pretty substantial change to how the dark theme overrides are applied (removal of the `:root` on the `:where(pf-v6-theme-dark`).

**Comment by mcoker** [comment]
`.page-loaded` isn't working consistently so opened https://github.com/patternfly/patternfly-org/issues/4009. Once that's complete, this issue should be unblocked.

**Comment by mcoker** [comment]
closed by https://github.com/patternfly/patternfly/pull/6692 and https://github.com/patternfly/patternfly/pull/6702
